mods/letsgo/scripts.js

```diff
@@ -47,7 +47,8 @@ let BattleScripts = {
 		if (nature.plus) stats[nature.plus] = Math.floor(stats[nature.plus] * 1.1);
 		// @ts-ignore
 		if (nature.minus) stats[nature.minus] = Math.floor(stats[nature.minus] * 0.9);
-		let friendshipValue = Math.floor((set.happiness / 255 / 10 + 1) * 100);
+		let friendshipValue = 100;
+		if (set.happiness != 0) friendshipValue += Math.floor(((set.happiness || 255) / 255 / 10) * 100);
```

Comment:
The build fails because you need two equal signs not one

Reply:
Honestly, this is all overcomplicated.

Reply:
Also this shouldn’t be `+=` it should just be overrided. With the +=  it becomes a 2.1 multiplier with max happiness. A simpler way would just be replacing the `set.happiness && set.happiness === 0 ? 0 : (set.happiness || 255)`

Reply:
The +1 was removed from Math.floor(((set.happiness || 255) / 255 / 10 + 1) * 100); so additive works just fine, but that definitely seems more concise. Sorry for the complications.